tools/src/bin/collect.rs

```diff
@@ -273,15 +312,46 @@ struct QueryOptions {
     current_batch: bool,
 }
 
+#[derive(Debug, Derivative, PartialEq, Eq, Subcommand)]
+enum Subcommands {
+    /// Create a new collection job and poll it to completion
```

Comment:
For the new code, it's for consistency with existing doccomments in collect.rs.

However, this style is inconsistent with the rest of Janus (spot checking janus_cli, and config.rs), so I will add punctuation to all help texts.

Reply:
I avoided periods at the end on structs used with Clap's derive macro because the periods got stripped off from the output of `--help`. At one point I put two sentences in a doc comment, and it looked funny with only one period in the middle. Experimenting with this some more now, it looks like Clap removes a trailing period if the doc comment is one line, but does nothing if there are multiple paragraphs. When passing `-h` instead of `--help`, then only the first paragraph is shown, and its trailing period is stripped.

Based on this, I think it would make sense to have the first line not include a period, try to keep the description brief so that it doesn't run on too long in the compressed `-h` format, and then use full sentences with periods below a blank line, so they don't appear in the short help, but appear as-is in long help.

Reply:
That seems reasonable. I'll adopt that approach for collect.rs and check the rest of Janus for consistency in another PR.